pkg/kubelet/kubelet_volumes.go

```diff
@@ -74,7 +74,7 @@ func (kl *Kubelet) ListBlockVolumesForPod(podUID types.UID) (map[string]volume.B
 // pods for the specified volume are mounted.
 func (kl *Kubelet) podVolumesExist(podUID types.UID) bool {
```

Comment:
maybe also update the comments mentioning about uncertain volumes?

Reply:
Fixed